storage/backup/backup-cli/src/coordinators/backup.rs

```diff
@@ -35,14 +35,33 @@ use tokio_stream::wrappers::IntervalStream;
 pub struct BackupCoordinatorOpt {
     #[clap(flatten)]
     pub metadata_cache_opt: MetadataCacheOpt,
-    // Taking a state snapshot involves going through the entire state tree, should be taken
-    // infrequently.
-    #[clap(long, default_value = "24")]
+    // Defaulting to 1 to try to always have the latest state snapshot.
+    #[clap(
+        long,
+        default_value = "1",
+        help = "Frequency (in number of epochs) to take state snapshots at epoch ending versions. \
+        Adjacent epochs share much of the state, so it's inefficient storage-wise and bandwidth-wise \
+        to take it too frequently. However, a recent snapshot is obviously desirable if one intends \
+        to recover a snapshot and catch up with the chain by replaying transactions on top of it. \
+        Notice that a snapshot won't be taken exactly according to this option if it's missed due \
+        to the previous backup finishing too late -- if the next epoch to be snapshoted is already \
+        available, the missed ones will be skipped, i.e. setting this to 1 will make the tool take \
+        a state snapshot at every epoch ending only if the previous epoch ending snapshot is finished \
+        before the one after the epoch isn't available yet."
+    )]
     pub state_snapshot_interval_epochs: usize,
-    // Assuming the network runs at 100 tps, it's 100 * 3600 = 360k transactions per hour, we don't
-    // want the backups to lag behind too much. Defaulting to 100k here in case the network is way
-    // slower than expected.
-    #[clap(long, default_value = "100000")]
+    // Defaulting to 1M, which converts to a 20 minutes delay of a transaction showing up in a backup,
+    // from a 1K TPS chain, and a few minutes replay time.
+    #[clap(
+        long,
+        default_value = "1000000",
+        help = "The frequency (in transaction versions) to take an incremental transaction backup. \
+        Making a transaction backup every 10 Million versions will result in the latest transaction \
+        to appear in the backup potentially 10 Million versions later. If the net work is running \
+        at 1 thousand transactions per second, that is roughly 3 hours. On the other hand, if \
+        backups are too frequent and hence small, it slows down loading the backup metadata by too \
+        many small files. "
```

Comment:
1. In the first sentence, it says  “will result in the latest txn to appear in the backup potentially 10 million versions later.” Is this correct? I think all we know is we will get incremental backup every 3 hours (~10 million txns), but we cannot say that the latest txn will be in the incremental backup, right?
2. But if we are taking the incremental backup starting with the latest finished txn and going backwards 10 million versions, then the above statement is correct.
3. **Suggested**: "The frequency (in transaction versions) of the incremental transaction backup. \
        For example, if this is set to 10 million and if the network is running 
        at 1 thousand transactions per second, then the incremental transaction backup is taken at roughly once in 3 hours. On the other hand, if \
        backups are too frequent and hence small, it slows down loading the backup metadata by too \
        many small files. Default value is 1000000 (one million)."